tools/installPluginVersion.php

```diff
@@ -55,6 +55,38 @@ function execute() {
 		$pluginVersion = $versionInfo['version'];
 		$versionDao = DAORegistry::getDAO('VersionDAO');
 		$versionDao->insertVersion($pluginVersion, true);
+
+		$pluginPath = str_replace('/version.xml', '', $this->_descriptor);
+		$pluginCategory = str_replace('plugins.', '', $pluginVersion->getProductType());
```

Comment:
Generally speaking I prefer using regular expression matches for this kind of thing -- they'll behave more predictably when the string doesn't match the pattern. It would probably be a good idea to check the result against a whitelist, since we're then including PHP from this name.